### PR TITLE
Updated for BT Dashboard deployment

### PR DESCRIPTION
**Pull request by Laurabraun1**
updated api to include BT project name and data columns. BT Issue #18

### CODE REVIEW

File: trails-viz-api/trailsvizapi/config/app_config.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
     # 'Mount Baker-Snoqualmie General Forest': 'MBS_GFA'
     'East Cascades': 'EastCascades',
     'Coronado': 'Coronado'
+    'Bridger Teton': 'BT'
```
Comment (sgwinder) [inline trails-viz-api/trailsvizapi/config/app_config.py:23]: I think Bridger-Teton should have a hyphen (here and in general)

File: trails-viz-api/trailsvizapi/config/app_config.py [hunk 2]
```diff
@@ -50,4 +51,5 @@
     # 'MBS_GFA': ['flickr', 'twitter', 'instag', 'wta', 'alltrails']
     'EastCascades': ['estimate', 'flickr', 'twitter', 'alltrails', 'ebird', 'wta', 'onsite'],
     'Coronado': ['estimate', 'flickr', 'twitter', 'alltrails', 'ebird', 'onsite']
+    'BT':['flickr','twitter','alltrails','ebird','onsite']
```
Comment (sgwinder) [inline trails-viz-api/trailsvizapi/config/app_config.py:54]: This will also need to include the name of the mobile data column.

**Review by sgwinder — CHANGES_REQUESTED** [review]
Thanks Laura! I left a couple of comments, and added @jblehr as a reviewer as well - Jacob I'll leave it up to you to test whether this works locally, and to incorporate any changes to the app required to add the mobile data.

File: trails-viz-api/trailsvizapi/config/app_config.py [hunk 3]
```diff
@@ -50,4 +51,5 @@
     # 'MBS_GFA': ['flickr', 'twitter', 'instag', 'wta', 'alltrails']
     'EastCascades': ['estimate', 'flickr', 'twitter', 'alltrails', 'ebird', 'wta', 'onsite'],
     'Coronado': ['estimate', 'flickr', 'twitter', 'alltrails', 'ebird', 'onsite']
```
Comment (jblehr) [inline trails-viz-api/trailsvizapi/config/app_config.py:53]: The `'Coronado'` line also needs a comma after it if you are adding a new line after (each line should end with a comma).

File: trails-viz-api/trailsvizapi/config/app_config.py [hunk 4]
```diff
@@ -20,6 +20,7 @@
     # 'Mount Baker-Snoqualmie General Forest': 'MBS_GFA'
     'East Cascades': 'EastCascades',
     'Coronado': 'Coronado'
```
Comment (jblehr) [inline trails-viz-api/trailsvizapi/config/app_config.py:22]: The 'Coronado' line also needs a comma after it if you are adding a new line after (each line should end with a comma).

**Comment by Laurabraun1** [comment]
Thanks @jblehr and @sgwinder - as always silly mistakes that shouldn't have happened. All fixed now! I added the mobile column as 'mobile' since I wasn't sure where we landed with gravy vs. mobile.

**Comment by sgwinder** [comment]
Thanks @Laurabraun1 ! 

As a note for @jblehr - the column in the data file is called `gravy`. I'm not sure whether that needs to match the argument in the `app_config.py` file?

**Comment by jblehr** [comment]
Noting here that this PR will also close https://github.com/OutdoorRD/trails-viz-data/issues/50.

**Review by jblehr — APPROVED** [review]
Noting here that I changed `app_config.py` to match the column name in the data: "gravy".

**Pull request merged**